### ISSUE TITLE
Git integration new/modified indicators only use color to convey meaning

### ISSUE THREAD

**Issue by Exalm** [elementary/files#1105]
## Prerequisites
- [x] I have searched open and closed issues for duplicates.

## Describe the bug

![Screenshot from 2019-11-14 14-50-30](https://user-images.githubusercontent.com/1510457/68845984-358cd480-06c4-11ea-8e30-b1f67a03b1be.png)

Green dot means new/unstaged, yellow means modified. The dots don't have tooltips or any label, so they can be hard to tell apart with a color blindness.

### To Reproduce
1. Open a git repo directory in Files
2. Create some files, modify some others

- [x] I'm using the latest version from git that I've manually compiled

**Comment by jeremypw** [comment]
Good point about accessibility. Using traditional tooltips is not possible as the emblemed file icons are not widgets, but maybe something could be shown in the overlay that appears on hovering?

**Comment by Exalm** [comment]
Dunno. I think `user-away` and `user-available` aren't correct semantically for this anyway (e.g. their symbolic variants outright don't make sense here with their chat bubble shape), both here and in Code, so maybe the icon set should have more specific icons for new/modified.

**Comment by Exalm** [comment]
See also: https://github.com/elementary/icons/issues/827

**Comment by GammaGames** [comment]
I think having a green `+` to show added files, yellow empty to indicate modified, and red `x` for deleted would make sense. I don't think I've seen any similar indicators across the system though, the closest might be the radio buttons with the checkbox or the notification bubbles in plank.

**Comment by jeremypw** [comment]
@GammaGames  That could be a solution - you should comment against https://github.com/elementary/icons/issues/837 where some other ideas have been offered.

**Comment by jeremypw** [comment]
I have marked this as "Upstream" because it is likely to be solved by changes to the icon theme first (although new emblem names would have to be used in Files).

**Comment by danirabbit** [comment]
![Image](https://github.com/user-attachments/assets/bfedc421-b2f1-4fca-9a1d-70151babbbcc)

These days we have git specific emblems but the "new" and "changed" emblem still only differ with color

**Comment by jeremypw** [comment]
@danirabbit Should this be transferred to `elementary/icons` as the problem seems to be a lack of suitable emblems to use?

**Comment by danirabbit** [comment]
No because adding non-freedesktop-namespaced icon names there is too fragile when someone can change their icon theme. I've been moving more icons into Granite, but that wouldn't be available to Files until after the Gtk4 port. And I'm not sure dealing with git status is a general enough thing to be worth maintaining there. It would probably be best to gresource icons here

**Comment by jeremypw** [comment]
OK, fair enough.  I am not an icon designer so I'll have to leave that to someone else.

**Comment by jeremypw** [comment]
Code also uses git emblems but probably nothing else that I am aware of.
